internal/user_data/internal/models/userData.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	"github.com/google/uuid"
+	"time"
+)
+
+type UserData struct {
+	UserID uuid.UUID `json:"user_id"`
+
+	Nickname string `json:"nickname"`
+
+	LastAuth time.Time `json:"last_auth"`
+
+	Registered time.Time `json:"registered"`
+
+	Avatar []string `json:"avatar"`
+
+	ContactList []string `json:"contact_list"`
+}
```

Comment:
```suggestion
type UserData struct {
	UserID      uuid.UUID `json:"user_id"`
	Nickname    string    `json:"nickname"`
	LastAuth    time.Time `json:"last_auth"`
	Registered  time.Time `json:"registered"`
	Avatar      []string  `json:"avatar"`
	ContactList []string  `json:"contact_list"`
}
```

All fields need to be aligned.